syntax = "proto3";

package response_models;
import "vector2d.proto";

message ShootingStateResponseSchema {
	int32 player_id = 1;
	int32 left_magazine = 2;
	int32 left_ammo = 3;
	bool is_reloading_required = 4;
	bool is_reloading = 5;
	util_models.Vector2D weapon_direction = 6;
}
